src/gromacs/nblib/simulationstate.cpp

```diff
@@ -65,10 +65,10 @@ namespace nblib
 
 SimulationState::SimulationState(const std::vector<gmx::RVec>& coord,
                                  Box                           box,
-                                 Topology&                     topo,
+                                 const Topology&               topology,
```

Comment:
The topology should be passed by value here and then passed on to topology_ with std::move.
This version always performs a copy, while the version that takes a value only copies if called with an lvalue and moves with an rvalue.

Reply:
Quite nifty to have CLion lint for recommending such changes.